vscode-trace-extension/src/extension.ts

```diff
@@ -6,15 +6,24 @@ import { TraceExplorerAvailableViewsProvider } from './trace-explorer/available-
 import { TraceExplorerOpenedTracesViewProvider } from './trace-explorer/opened-traces/trace-explorer-opened-traces-webview-provider';
 import { fileHandler, openOverviewHandler, resetZoomHandler, undoRedoHandler, zoomHandler, keyboardShortcutsHandler } from './trace-explorer/trace-tree';
 import { TraceServerConnectionStatusService } from './utils/trace-server-status';
-import { updateTspClient } from './utils/tspClient';
+import { updateTspClient, getTspClientUrl } from './utils/tspClient';
 import { TraceExtensionLogger } from './utils/trace-extension-logger';
 import { VSCODE_MESSAGES } from 'vscode-trace-common/lib/messages/vscode-message-manager';
+import { TspClientProvider } from 'vscode-trace-common/lib/client/tsp-client-provider-impl';
 
 export let traceLogger: TraceExtensionLogger;
+let tspClientProvider: TspClientProvider;
```

Comment:
As far as I can tell, this could just be initialized immediately, since it doesn't depend on anything that's passed into the `activate` function, and that would save some checks for whether it's defined.

Reply:
I'll check this.